lib/bucky/test_equipment/user_operation/user_operation_helper.rb

```diff
@@ -33,8 +35,13 @@ def clear(args)
         def click(args)
           elem = @pages.get_part(args)
           elem.location_once_scrolled_into_view
-          sleep 1
-          elem.click
+          wait = Selenium::WebDriver::Wait.new(:timeout => 5, :interval => 0.1, :ignore => [Selenium::WebDriver::Error::WebDriverError])
+          begin
+            # it will return nil if click successfully
+            wait.until{ elem.click.nil? }
+          rescue Selenium::WebDriver::Error::TimeoutError
+            raise Selenium::WebDriver::Error::TimeoutError, "Exceeded the limit for trying to click.\n    #{$ERROR_INFO.message}"
```

Comment:
```suggestion
            raise Selenium::WebDriver::Error::WebDriverError, "Exceeded the limit for trying to click.\n    #{$ERROR_INFO.message}"
```

Reply:
I think this is easier to understand the error information than TimeoutError.
How about you?

Reply:
Yes, I think so too.
I'll fix the spec also.